src/app/app.component.ts

```diff
@@ -13,4 +13,4 @@ export class AppComponent {
   toggleSidebar(): void {
     this.sideBarShown = !this.sideBarShown;
   }
-}
+}
```

Comment:
You are actually supposed to leave an extra newline at the end of a file. It is helpful for some operating systems or something. Since some of them expect an empty line at the end of a file. So generally you are supposed to have an empty line at the end of files.

That is why there is a red icon on this line in the files changed section

Reply:
Also the lack of a new line at the end of the file is causing the linter to fail so Travis CI fails.

Also there are code smells that are causing SonarCloud to fail. So you will have to fix those as well